packages/ComponentLibrary/src/components/Table/styles.ts

```diff
@@ -74,4 +67,58 @@ export const sx: { [key: string]: SxProps<Theme> } = {
       textDecoration: 'underline',
     },
   },
+  leftSection: {
+    background: theme.palette.baselineColor.neutral[0],
+    display: 'flex',
+    padding: '0.25rem',
+    gap: '0.25rem',
+    borderRadius: '6.25rem',
+  },
+  newLineButton: {
+    background: theme.palette.baselineColor.neutral[100],
+    width: '9.375rem',
+    borderRadius: '6.25rem',
+    padding: '0.25rem 0 0.25rem 0.75rem',
+    justifyContent: 'flex-start',
+    paddingLeft: '0.75rem',
+    color: theme.palette.baselineColor.neutral[0],
+    '&:hover': {
+      border: 'none',
+      background: theme.palette.dynamicColor.main,
+      borderRadius: '6.25rem',
+    },
+  },
+  sidebarContainer: {
+    height: '100%',
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  headerContainer: {
+    display: 'flex',
+    alignItems: 'start',
+    gap: 2,
+    padding: '1rem',
+  },
+  iconContainer: {
+    flexShrink: 0,
+    height: '3rem',
+    width: '3rem',
+    background: theme.palette.baselineColor.neutral[0],
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    borderRadius: '50%',
```

Comment:
Really, this is not good :|
It is not good practice to use percentages in border radius